.eslintrc

```diff
@@ -0,0 +1,91 @@
+{
+    "env": {
+        "browser": true,
+        "jasmine": true,
+        "amd": true,
+        "node": true
+    },
+    "globals": {},
+    "rules": {
+        "no-bitwise": 2,
+        "curly": [
+            2,
+            "all"
+        ],
+        "eqeqeq": 2,
+        "no-eq-null": 2,
+        "guard-for-in": 2,
+        "no-extend-native": 2,
+        "no-use-before-define": 2,
+        "complexity": [
+            2,
+            15
+        ],
+        "no-caller": 2,
+        "no-irregular-whitespace": 2,
+        "no-undef": 2,
+        "no-unused-vars": 2,
+        "space-return-throw-case": 0,
+        "space-after-keywords": [
```

Comment:
i would update this as it will break in the next eslint version. https://github.com/agrc/AGRCJavaScriptProjectBoilerPlate/commit/ef98a3e4cc22fa2877dc9ea968733bec925aaae9